packages/utils/dropdown/ResourceDropdown.tsx

```diff
@@ -218,6 +220,7 @@ const ResourceDropdown: ResourceDropdown = <T extends unknown>({
           onToggle={loaded && !loadError ? onToggle : () => {}}
           toggleIndicator={CaretDownIcon}
           data-test={`${dataTest}-toggle`}
+          isDisabled={disabled}
```

Comment:
looks like this will not work as we expected !!
can u quickly test it once ?
I think even if `DropdownToggle` is disabled it will still show `Error Loading` message that we have below if there is any error... and we do not want that... that is not good... if that if not happening then we r okay... else we need to go back to previous approach (that do not look good but atleast works fine) !

Reply:
How to make it fail, so that we can test the error scenario

Reply:
login as a user who do not have access to a namespace...

Reply:
it will not work I checked here: https://www.patternfly.org/v4/components/dropdown#disabled
that is how DropdownToggle works... not in our control I guess...

Reply:
rolling back to our previous approach of dropdown? right?

Reply:
yeah